xm-commons-messaging/src/main/java/com/icthh/xm/commons/messaging/communication/CommunicationMessage.java

```diff
@@ -33,6 +37,28 @@ public class CommunicationMessage {
     private String version;
     private List<Receiver> receiver = new ArrayList<>();
     private Sender sender;
-    private List<CommunicationRequestCharacteristic> characteristic;
+    private List<CommunicationRequestCharacteristic> characteristic = new ArrayList<>();
+
+    public CommunicationMessage setTemplateModel(String templateModel) {
```

Comment:
create separate builder without modification of this class

Reply:
Created separate class